src/services/users/users.ts

```diff
@@ -49,7 +49,7 @@ export default class UserSdk {
 
   createUser(identityToken: string): Promise<CreateUserResponse> {
     return this.axiosClient.post<CreateUserResponse>(
-      `/users`,
+      `/users/users`,
```

Comment:
`users/users`? LOL

Reply:
Bahahah yeah... its an old service LOL

Reply:
Its going to be refactored in the future, but for now this is what we got 😂